lib/drab/config.ex

```diff
@@ -247,7 +247,11 @@ defmodule Drab.Config do
       config :drab, disable_controls_while_processing: false
   """
   @spec get(atom) :: term
-  def get(:templates_path), do: Application.get_env(:drab, :templates_path, "priv/templates/drab")
+  def get(:templates_path) do
+    Application.get_env(:drab, :app) && 
```

Comment:
I would use `Drab.Config.app_name/0` instead introducing the new config option, `:app` (it is redundant for existing `:main_phoenix_app` [which we should rename]).

Also, if someone does `config :drab, templates_path: "..."` without the `:app`, `app_dir` would fail with the confusing message.

Reply:
Yeah. Another concerns is that in an umbrella project you could have more than one web app, in this case we cannot stands on  `:main_phoenix_app`, because though each app have it own drab configuration, in an Umbrella project the apps configurations are merged and conflicting keys overridden, so the configuration retrieved is not the expected one.

To circumvent this, we should pass the app name as an option in `Drab.Modal.alert/4` instead to reading it from the configuration, this way there will be the right path also when Drab is used in different umbrella apps, but this require a substantial modification in `Drab.Config` and maybe in other modules.

What do you think about?

Reply:
Yep.
I've never thought about multiple drabs in umbrella, but why not? Should be allowed. It will require changes in the config API, for example something like:

config :drab, MyAppWeb, templates_path: "xxx"

I will open additional issue for this.